public/main.css

```diff
@@ -0,0 +1,157 @@
+/*****Header*****/
+
+.header {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: 250px;
+    background-color: #a9d2de;
+}
+
+#fish {
+    display: grid;
+   grid-column: 1 /5;
+    border: 4px solid darkgray;
+    border-radius: 5px;
+}
+
+.restBox {
+    display: flex;
+    height:auto;
+    text-align: center;
+    text-decoration: none;
+    border: 4px solid darkgray;
+    border-radius: 5px;
+}
+
+
+
+/*****A box to hold things ******/
+
+.container {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: auto;
+    overflow-x: hidden;
+    border-top: 1px solid #DADCE0;
+    border-bottom: 1px solid #DADCE0;
+    background-color: #a9d2de;
+}
+
+#mapid {
+    display: flex;
+    grid-column: 1/ 5;
+    height: 500px;
+    border: 4px solid darkgray;
+    border-radius: 5px;
+}
+
+/******** Styling information for Footer*****/
+
+.footerContainer {
+    display: grid;
+    grid-column-start: 1;
+    grid-column-end: 6;
+    order: 7;
+    background-color: #a9d2de;
+}
+
+.footer {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: repeat(2, 5px) repeat(2, 1fr);
+    grid-gap: 1rem;
+    grid-template-areas: "footerTitle       footerTitle        footerTitle      footerTitle" "formSection       formSection        footerText       footerText" "formSection       formSection        footerAddress    footerTwitter" "formSection       formSection        footerPhone      footerWebsite" "formSection       formSection        footerEmail      footerFacebook" "formSection       formSection        .   footerInstagram";
```

Comment:
It looks like there is some funkiness happening here with your grid. You have defined a four by four grid ( four 1fr columns, by two 5px rows and two 1fr rows), but your grid template area is a four column by six row template. Your template should match up with the number of columns and rows you define above.